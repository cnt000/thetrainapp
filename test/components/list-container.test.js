import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import ListContainer from '../../src/components/list-container'

describe('ListContainer', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<ListContainer />, div)
  })

  test('has a valid snapshot', () => {
    const component = renderer.create(<ListContainer />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
