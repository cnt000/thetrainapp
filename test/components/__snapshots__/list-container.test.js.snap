// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListContainer has a valid snapshot 1`] = `
<div
  className="sc-bdVaJa hgwxvP"
/>
`;
